web/themes/custom/pineapple/templates/content/node--recipe--full.html.twig

```diff
@@ -101,17 +101,18 @@
     </div>
     <aside class="recipe-sidebar">
       <div class="recipe-author-container">
-        {% set author_image_url = bamboo_render_image_style(content.field_author|field_value|first['#node'].getFields().field_a_image.entity.field_media_image.entity.getFileUri(), 'thumbnail_115_x_115') %}
-        {% set author_image_alt = content.field_author|field_value|first['#node'].getFields().field_a_image.entity.field_media_image.alt %}
+        {% set author_node = content.field_author|field_value|first['#node'] %}
+        {% set author_image_url = bamboo_render_image_style(author_node.getFields().field_a_image.entity.field_media_image.entity.getFileUri(), 'thumbnail_115_x_115') %}
```

Comment:
I’m pretty sure you don’t need to do author_node.getFields().field_name you should be able to just do author_node.field_name

Reply:
done